### ISSUE TITLE
Dataset code

### ISSUE THREAD

**Issue by thibautloiseau** [ByteDance-Seed/Depth-Anything-3#61]
Hi, thanks for the code and the paper, great work!
Do you plan on releasing the code for the different datasets you used? That would be very useful as there are many different datasets you used for training the model.
Thanks a lot!

**Comment by hzzzzzhappy** [comment]
Good job! I would like to know whether the training data or data loader will be released. Thx author.
